ui/src/app/shared/get_workflow_params.ts

```diff
@@ -0,0 +1,37 @@
+import {History} from 'history';
+/**
+ * Method to extract a key from query parameter. The query parameter will be of the following format
+ * ?parameters[key]=value.
+ * This method will extract the key from the query parameter.
+ * @param inputString
+ * @returns
```

Comment:
This repository usually doesn't have JSDoc annotations since it uses TypeScript:
```suggestion
```